src/core/api/scan_all.go

```diff
@@ -68,9 +67,57 @@ func (sc *ScanAllAPI) Post() {
 		sc.SendBadRequestError(err)
 		return
 	}
-	ajr.Name = common_job.ImageScanAllJob
-	sc.submit(&ajr)
-	sc.Redirect(http.StatusCreated, strconv.FormatInt(ajr.ID, 10))
+
```

Comment:
the api will migrate in the new program model as well?

Reply:
This may in the next PR.